src/NodeProcess.cs

```diff
@@ -12,6 +13,12 @@ internal class NodeProcess
 
         private static string _installDir = Path.Combine(Path.GetTempPath(), Vsix.Name, Packages.GetHashCode().ToString());
         private static readonly string _executable = Path.Combine(_installDir, "node_modules\\.bin\\prettier.cmd");
+        private static readonly string[] _supportedExtensionRegexes = new string[] { ".jsx?", ".tsx?" };
```

Comment:
Do you use this anywhere?

Reply:
Huh.. I thought I had removed that change.  That's for adding tslint support (another issue)

Reply:
Oops.. thought I had removed that.  I'll update